### ISSUE TITLE
Okta signout only works on the home page for okta-auth-js 5.9.1

### ISSUE THREAD

**Issue by mjackson6** [okta/okta-auth-js#1041]
Hi,

   I have been working with okta authentication using angular and I have been seeing inconsistencies with the signout() method. When I call `await this.oktaAuth.signOut();` from the home page, http://localhost:4200/, the signout is successfully. However, when I call that same method from another page, i.e http://localhost:4200/about, its fails to signout the user out of okta, but does revoke the tokens. 

**Here is the network screen shot from Chrome devtools network**

![network screen shot](https://user-images.githubusercontent.com/96541175/147128252-c427bf7f-8f12-44fe-a84c-909f9b520e19.png)

**And here is the error that is returned.** 

![authapierror](https://user-images.githubusercontent.com/96541175/147128842-04decf0e-ea9d-418c-90a1-97f7a5b6adbd.png)

Is there a restriction where the signout() only works from the home page?

Additional information
    "@okta/okta-angular": "^5.0.0",
    "@okta/okta-auth-js": "^5.9.1", 

    http://localhost:4200 is added to the Trusted Origins in the Admin console.

**Comment by shuowu** [comment]
@mjackson6 The signOut method should not be affected by routes. From your network logs, it looks like the token revocation requests was cancelled by some reason. Maybe try call the method with following options as a workaround:
```
this.oktaAuth.signOut({
  revokeAccessToken: false,
  revokeRefreshToken: false
});
```
to skip the token revocation logic?

You can also try [Okta's Angular sample apps](https://github.com/okta/samples-js-angular) (I failed to reproduce the issue there..)

**Comment by mjackson6** [comment]
Hi @shuowu,

   Thank you for your response. I tried your suggestion and I ran into a similar issue, except this time, the /logout url is cancelled

![network screen shot2](https://user-images.githubusercontent.com/96541175/147140363-714d766a-73f1-4d1e-a5e7-73e90ff7925a.png)


Here is what is looks like on admin console

![admin shot](https://user-images.githubusercontent.com/96541175/147140760-1ee4e42d-7312-4708-a07b-06710ffde744.png)


And then here is the successful signout from the http://localhost:4200/.

![success network](https://user-images.githubusercontent.com/96541175/147141146-9b0f9a4b-9de8-4901-a7d2-97b2a441f4ba.png)



Is there any reason why the /revoke or /logout endpoints would cancel?

**Comment by shuowu-okta** [comment]
@mjackson6 I found [this thread](https://stackoverflow.com/a/24062160), might be relevant.

**Comment by mjackson6** [comment]
Thanks for your help @shuowu I found out what was causing the issue for this. We were using the following link to called the signout function

    <a href="#" (click)="clickLink($event, 'signout')">

The href="#" redirects to the top of the current page. This would work on the home page since the redirectUrl was the same. However, on a different page, this would cause the fetch to be cancelled. To resolve the issue, in the clickLink() function, we added event.preventDefault(). This cancelled the default behavior for href. See below

clickLink(event: MouseEvent, value: string) {
    // Prevent the default behavior of href.
    event.preventDefault();
  }
